server/routes/viewingLicences/handlers/printLicence.ts

```diff
@@ -39,16 +39,24 @@ export default class PrintLicenceRoutes {
       user
     )
 
-    const additionalConditions = licence.additionalLicenceConditions.filter(
-      (c: AdditionalCondition) => !additionalConditionsWithUploads.find((c2: AdditionalCondition) => c.id === c2.id)
-    )
+    const getGroupedAdditionalConditions: Map<string, AdditionalCondition[]> =
```

Comment:
some weird renaming going on here?

You have two impls of `getGroupedAdditionalConditions`?

Reply:
fixed, only worked because one of them was a method